nbs/core.ipynb

```diff
@@ -54,10 +54,15 @@
     "#| export\n",
```

Comment:
<p>Could we add unit tests for the <code>save/load</code> functionality? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done